lib-ce/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -206,11 +206,15 @@ is_valid_pwd(<<Salt:4/binary, Hash/binary>>, Password) ->
 %%--------------------------------------------------------------------
 
 init([]) ->
-    %% Add default admin user
-    {ok, _} = mnesia:subscribe({table, mqtt_admin, simple}),
-    PasswordHash = ensure_default_user_in_db(binenv(default_user_username)),
-    ok = ensure_default_user_passwd_hashed_in_pt(PasswordHash),
-    ok = maybe_warn_default_pwd(),
+    case binenv(default_user_username) of
+        <<>> -> ok;
+        UserName ->
+            %% Add default admin user
+            {ok, _} = mnesia:subscribe({table, mqtt_admin, simple}),
```

Comment:
We should always subscribe mnesia?

Reply:
in case some other node has a different config

Reply:
it's not necessary to subscribe mnesia if the local default is missing.

because we do:
```
handle_info({mnesia_table_event, {write, Admin, _}}, State) ->   
    #mqtt_admin{username = Username, password = HashedPassword} = Admin,
    case binenv(default_user_username) of // always empty if default is missing, we don't change anything.
        Username ->
            ok = ensure_default_user_passwd_hashed_in_pt(HashedPassword);
        _ ->
            ignore
    end,
    {noreply, State};
```